<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="RGBGroup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>171</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>RGB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>G:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="redText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="greenText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="blueText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="HSVGroup">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>171</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>HSV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>H:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>S:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>V:</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="hueText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="saturationText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="valueText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LABGroup">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>171</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>LAB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>L:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>A:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-128</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-128</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="lText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="aText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-128</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="bText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-128</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>671</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>valueText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>HUEChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>HUEChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RGBChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>LABChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>HUEChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valueText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redText</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RGBChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RGBChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>LABChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aText</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>LABChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButtonClicked()</slot>
  <slot>RGBChanged()</slot>
  <slot>HUEChanged()</slot>
  <slot>LABChanged()</slot>
 </slots>
</ui>
